Dockerfile

```diff
@@ -1,28 +1,28 @@
-FROM openshift/origin-base
-
-ENV APP_HOME=/opt/app/src \
-    GOPATH=/go \
-    PATH="${PATH}:${GOPATH}/bin:/usr/local/bin" \
-    INSTALL_PKGS="golang make git"
+# first stage
+FROM openshift/origin-release:golang-1.10 AS builder
 
-RUN yum install -y ${INSTALL_PKGS} --setopt=tsflags=nodocs
+ENV GOPATH=/go
 
 WORKDIR /tmp/cluster-logging-operator
 COPY vendor/ /tmp/cluster-logging-operator/vendor
-COPY scripts/* /usr/local/bin/scripts/
 COPY cmd/ /tmp/cluster-logging-operator/cmd
 COPY pkg/ /tmp/cluster-logging-operator/pkg
 COPY Makefile /tmp/cluster-logging-operator
 
-RUN make && \
-    cp _output/bin/cluster-logging-operator /usr/local/bin/ && \
-    rm -rf /tmp/cluster-logging-operator && \
-    yum erase -y ${INSTALL_PKGS} && \
-    yum clean all
+RUN make
+
+# second stage
+
+FROM openshift/origin-base
+
+ENV PATH="${GOPATH}/bin:/usr/local/bin:${PATH}"
```

Comment:
FWIW `$GOPATH` isn't defined here so this is probably not doing anything. That first segment should probably be removed.